ms/MSSel/MSSpwGram.cc

```diff
@@ -130,18 +130,16 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
 			     Vector<Int>& selectedDDIDs) 
   {
     Int ret;
-    //    MSSpwParse *thisParser = new MSSpwParse(ms);
-    MSSpwParse *thisParser = new MSSpwParse(spwSubTable, ddSubTable, colAsTEN);
+    MSSpwParse thisParser(spwSubTable, ddSubTable, colAsTEN);
     try
```

Comment:
There is no purpose for this try since you only rethrow the exception. The code can simply be:
    MSSpwParse thisParser(spwSubTable, ddSubTable, colAsTEN);
    return baseMSSpwGramParseCommand(&thisParser, command, selectedIDs, selectedChans, selectedDDIDs);